src/Routing/QueryRouteEnhancer.php

```diff
@@ -29,11 +29,8 @@ public function enhance(array $defaults, Request $request) {
     $query = $this->extractQuery($request);
     $operations = $helper->parseRequestParams($method, $body, $query);
 
-    // By default we assume a 'single' request. This is going to fail in the
-    // graphql processor due to a missing query string but at least provides
-    // the right format for the client to act upon.
     return $defaults + [
-      '_controller' => $defaults['_graphql']['single'],
+      '_controller' => $defaults['_controller'],
```

Comment:
What is reason if this line? I see it does nothing useful.

Reply:
For me, the change fixes a Notice
```
Notice: Trying to access array offset on value of type bool in Drupal\graphql\Routing\QueryRouteEnhancer->enhance() (line 36 of /app/drupal/web/modules/contrib/graphql/src/Routing/QueryRouteEnhancer.php).
```

Reply:
I've applied this patch and it seems issue is solved and everything works (tried to run all our queries).